### ISSUE TITLE
Bug: Opening database without metadata causes freeze

### ISSUE THREAD

**Issue by wmeijer221** [ArchitecturalKnowledgeAnalysis/EmailDatasetBrowser#13]
I just tried to import your dataset ``iteration-7`` into EmailDatasetBrowser 2.5.3, which froze the application.

I saw that the dataset didn't have the ``metadata.properties`` file, for which (I think) the application generates one and sets the ``version`` to ``1``. When I changed this manually to ``2`` the dataset could load.

Upgrading the dataset to version 2 causes the app to freeze as well, so I think there's a bug in the upgrade path. (Because the dataset is already version 2, whereas its properties still say its version 1, so it's trying to do something impossible.)

If you simply make sure that the ``metadata.properties`` file exists in all of your exports with the correct ``version`` property, it works as well.

**Comment by andrewlalis** [comment]
Yes, I noticed this during my meeting with Mohamed as well lol. It'll be fixed in a few moments.

**Comment by andrewlalis** [comment]
Fixed with release 2.5.4: https://github.com/ArchitecturalKnowledgeAnalysis/EmailDatasetBrowser/releases/tag/v2.5.4

Sorry it took so long, but apparently Jitpack, which I use for building the dependencies of the browser app, supports jdk 17.0.2 but not jdk 17.0.3, and I only found out by scouring the build logs.
